crates/turbopack-node/src/lib.rs

```diff
@@ -200,9 +201,9 @@ pub async fn get_renderer_pool(
     )
     .await?;
 
-    emit(intermediate_asset, intermediate_output_path).await?;
+    emit(intermediate_asset, output_root).await?;
 
-    let cwd = intermediate_output_path.root();
+    let cwd = output_root;
```

Comment:
This is completely broken, though to be fair it was partially broken before (workspaces wouldn't have worked)